app/models/course/assessment/assessment_ability.rb

```diff
@@ -159,4 +161,12 @@ def allow_students_read_own_assessment_answers
   def allow_staff_read_assessment_answers
     can :read, Course::Assessment::Answer, discussion_topic: course_staff_hash
   end
+
+  def allow_students_read_submission_question
+    can :read, Course::Assessment::SubmissionQuestion, submission: { creator_id: user.id }
```

Comment:
This syntax should be checking for the creator of the submission, not the creator of the submission_question.